lib/task-data/schemas/types-installos.json

```diff
@@ -319,6 +319,22 @@
             "type": "string",
             "description": "Extra (persistent) kernel boot parameters",
             "minLength": 1
+        },
+        "PackagesArray": {
+            "description": "specify additional packages to install",
+            "type": "array",
+            "items": {
+                "type": "string"
+            },
+            "uniqueItems": true
+        },
+        "ClearDisk": {
+            "description": "a flag to indicate whether partitions of non-installation should be cleared",
+            "type": "boolean"
+        },
+        "Kdump": {
```

Comment:
add a field 'Kdump' is not a general way, if others want to add other items want to disable/enable by systemctl, more fields will be added, it's not so good.  so either controlling it by puppet/ansible/.. after os installation, or adding a more general way for things which could be controlled by 'systemctl'

Reply:
Thanks for the suggestion. Removed kdump from the PR. Will add support for providing the list of services that needs to be enabled / disabled during the post-installation.